src/gui/qgsattributeform.cpp

```diff
@@ -2129,16 +2131,26 @@ QgsAttributeForm::WidgetInfo QgsAttributeForm::createWidgetFromDef( const QgsAtt
       const QgsAttributeEditorQmlElement *elementDef = static_cast<const QgsAttributeEditorQmlElement *>( widgetDef );
 
       QgsQmlWidgetWrapper *qmlWrapper = new QgsQmlWidgetWrapper( mLayer, nullptr, this );
+
+      qmlWrapper->setResizeFlag( elementDef->resize() );
       qmlWrapper->setQmlCode( elementDef->qmlCode() );
+
       context.setAttributeFormMode( mMode );
       qmlWrapper->setContext( context );
 
       mWidgets.append( qmlWrapper );
 
-      newWidgetInfo.widget = qmlWrapper->widget();
+      // newWidgetInfo.widget = qmlWrapper->widget();
+      QgsScrollArea *scrollArea = new QgsScrollArea( parent );
+      scrollArea->setWidgetResizable( true );
+      scrollArea->setMinimumWidth( 400 );
```

Comment:
I see the need for this ScrollArea and it's size. But this is very static. It's always 400 or am I wrong?
If we cannot have it dynamic according the size of the form, then maybe this should be configurable as well in the properties. What do you think?

Reply:
The widget wrapper must be wrapped. If not, the expressions are not calculated. I didn't want to add this additional ScrollArea. I just added it to avoid the error with expressions in the feature form.

It works, but I still get the error in the console.

The scroll area is not taking the all area. I've added this minimum width/height just for debug.

Reply:
I've removed this minimum width and height. It just takes all the horizontal space, not the vertical one.